redhawk/python/python_tree_converter.py

```diff
@@ -593,50 +613,70 @@ def ConvertTryfinally(self, tree):
                      body = self.ConvertListOfStatements(tree.body),
                      final_body = self.ConvertListOfStatements(tree.finalbody))
 
+  def ConvertTry(self, tree):
+    # New AST type in Python 3
+    # TODO: is this correct?
+    return N.Try(position=self.gc.GC(tree),
+                 body = self.ConvertListOfStatements(tree.body),
+                 exception_handlers = list(map(self.ConvertTree, tree.handlers)),
+                 orelse = self.ConvertListOfStatements(tree.orelse),
+                 final_body = self.ConvertListOfStatements(tree.finalbody))
 
   def ConvertWith(self, tree):
     """ Convert With(expr context_expr, expr? optional_vars, stmt* body) node."""
-    assign_node = ast.Assign(
+    if hasattr(tree, 'optional_vars'):
```

Comment:
Ok, I'll add TODO lines to places where I changed current methods because of Python 3 compatibility.